src/secrets-kubernetes/src/lib.rs

```diff
@@ -106,6 +92,45 @@ impl SecretsController for KubernetesSecretsController {
             )
             .await?;
 
+        // guarantee that all new secrets are reflected on our local filesystem
+        let secrets_storage_path = PathBuf::from("/secrets");
```

Comment:
Are you sure you want to hard-code this string in here?

Reply:
Yes, please pass this in as a command-line argument.